pkg/storage/etcd/etcd_watcher.go

```diff
@@ -359,9 +359,6 @@ func (w *etcdWatcher) sendAdd(res *etcd.Response) {
 		return
 	}
 	action := watch.Added
-	if res.Node.ModifiedIndex != res.Node.CreatedIndex {
```

Comment:
@hongchaodeng please add a release note to this issue, as there is the behavioral change.